Dockerfile

```diff
@@ -1,8 +1,10 @@
-FROM golang:1.8 AS build
+ARG GO_VERSION=1.17
+
+FROM golang:${GO_VERSION} AS build
 COPY . /go/src/github.com/cpuguy83/go-md2man
 WORKDIR /go/src/github.com/cpuguy83/go-md2man
-RUN CGO_ENABLED=0 go build
+RUN make build
```

Comment:
Wondering; was there a reason to continue using the makefile for this? If there's no more uses for `BUILD_FLAGS`, I'm wondering if a plain `go build` would work (if needed with  a `-o` to store the binary somewhere else). Looks like building is almost instant, so we could even run `go test` as part of the build, and run a `md2man --version` to verify the binary is not defunct

(edit: erm, except it doesn't have a `--version` or `--help` 🤔 )

Reply:
No reason other than to make sure its the same command.